test/checkpoint/checkpointer.c

```diff
@@ -32,22 +32,22 @@ static WT_THREAD_RET checkpointer(void *);
 static WT_THREAD_RET clock_thread(void *);
 static int compare_cursors(WT_CURSOR *, table_type, WT_CURSOR *, table_type);
 static int diagnose_key_error(WT_CURSOR *, table_type, int, WT_CURSOR *, table_type, int);
-static int real_checkpointer(void);
+static int real_checkpointer(THREAD_DATA *);
 
 /*
  * set_stable --
  *     Set the stable timestamp from g.ts_stable.
```

Comment:
Can we update this description of the `set_stable` function? Could be confusing as this describes the old behavior.

Reply:
Fixed.